conu/backend/docker/skopeo.py

```diff
@@ -0,0 +1,65 @@
+"""
+Wrapping skopeo's functionality
+"""
+
+from enum import Enum
+from conu.exceptions import ConuException
+
+
+class Transport(Enum):
```

Comment:
nit: SkopeoTransport could be a more clear name; @rpitonak WDYT?

Reply:
+1